cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -754,14 +754,22 @@ label:updated[]
 SHOW DATABASES YIELD *
```

Comment:
should we add/update so this isn't just with YIELD?

Reply:
I removed the YIELD *